lib/rbui/textarea/textarea.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module RBUI
+  class Textarea < Base
+    def initialize(rows: 4, **attrs)
```

Comment:
I've changed to 4 because shadcn uses it

Reply:
![image](https://github.com/user-attachments/assets/7428b5d3-0e0e-43b0-b38c-3b1ef4408b01)